### ISSUE TITLE
Attempt to create PowerShell remote session throw error in version 7.3+

### ISSUE THREAD

**Issue by travisgan** [PowerShell/PowerShell#19130]
### Prerequisites

- [X] Write a descriptive title.
- [X] Make sure you are able to repro it on the [latest released version](https://github.com/PowerShell/PowerShell/releases)
- [X] Search the existing issues.
- [X] Refer to the [FAQ](https://github.com/PowerShell/PowerShell/blob/master/docs/FAQ.md).
- [X] Refer to [Differences between Windows PowerShell 5.1 and PowerShell](https://docs.microsoft.com/powershell/scripting/whats-new/differences-from-windows-powershell).

### Steps to reproduce

Enter-PSSession -Computer ComputerA -ConfigurationName powershell.7

### Expected behavior

```console
Enter into interactive PowerShell (PowerShell 7) remote session in ComputerA
```

### Actual behavior

```console
It could not establish a PowerShell remote session and throw the following error.
```

### Error details

```console
Enter-PSSession: Connecting to remote server ComputerA failed with the following error message : Could not load file or assembly 'System.Security.Cryptography, Version=7.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'. The system cannot find the file specified. For more information, see the about_Remote_Troubleshooting Help topic.
```

### Environment data

```powershell
Tested with the target server installed with PowerShell version 7.3.1 and 7.3.2, and remote session created from a workstation/server with PowerShell version 7.2.4, 7.2.6, 7.2.9, 7.3.1, 7.3.2.

When the target server is version 7.3+, it throws this error.

It works fine when the target server is version 7.2.4, 7.2.6, 7.2.9.
```

**Comment by rhubarb-geek-nz** [comment]
I have both 7.3.2 on local and remote machine and using SSH as the transport.

```
C:\Users\somebody>pwsh
PowerShell 7.3.2
PS C:\Users\somebody> Enter-PSSession -HostName 10.1.2.15
[10.1.2.15]: PS C:\Users\somebody\Documents> $PSVersionTable

Name                           Value
----                           -----
PSVersion                      7.3.2
PSEdition                      Core
GitCommitId                    7.3.2
OS                             Microsoft Windows 10.0.22621
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0…}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0

```

**Comment by travisgan** [comment]
The issue experienced is when using WinRM (not SSH) on Windows into PowerShell 7 session (also note, not Windows PowerShell 5 session which is still working). Initiating from Windows 10 workstation/Windows Server 2019, to remote to Windows Server 2019 encountered the error.

Enter-PSSession -Computer ComputerA -ConfigurationName powershell.7

**Comment by daxian-dbw** [comment]
The reported issue was fixed in PowerShell v7.3.2. See the verification of the fix here: https://github.com/PowerShell/PowerShell/issues/18587#issuecomment-1421862985
